data/ka'het/ka'het missions.txt

```diff
@@ -86,12 +96,60 @@ mission "First Contact: Ka'het: Remnant 1B"
 			choice
 				`	"Wait, why did you call them 'shells' and not ships?"`
 			label shells
-			`	"You see, <first>," she says, "the Ka'het aren't normal spaceships. The debris we found were artificial exoskeletons, built around huge living beings; it's unlikely a species that big could have developed that sort of technology, so we presume someone must have created it for them. This could have happened in the Graveyard, since it's the only place where we know them to live in, but we have such limited data about that section of space that we can't say for sure." As she finishes talking, one of the people Taely was talking to earlier comes closer and shows her something on a touchpad. "I have to go now, Captain," she tells you. "Good luck."`
+			`	"You see, <first>," she says, "the Ka'het aren't normal spaceships. The debris we found were artificial exoskeletons, built around huge living beings; it's unlikely a species that big could have developed that sort of technology, so we presume someone must have created it for them. We call them the Builders. After we lost the first scout ships in the Graveyard, the prefects decided that we should continue to scout the region but take necessary precautions. Thanks to those protocols, some of which are still applied today, we were able to extensively explore most of the Graveyard and decipher a good amount of the written language we found. There is still much we have yet to discover." You discuss the Graveyard with Taely for a while longer until one of the people she was talking to earlier comes closer and shows her something on a touchpad. "I have to go now, Captain," she tells you. "Good luck."`
 				decline
 	on decline
+		event "Graveyard insights"
 		log `Asked Taely about the Graveyard and the Ka'het and learned of the Remnant's previous contacts between them.`
 		log "Factions" "Ka'het" `The Ka'het are a species which live in the Graveyard, an area south of the Ember Waste. They are large lifeforms which can travel in space using artificial exoskeletons and will attack anyone on sight, even other Ka'het.`
 
+mission "Graveyard Insights Compatibility"
+	landing
+	invisible
+	to offer
+		has "First Contact: Ka'het: Remnant 1B: offered"
+		not "event: Graveyard insights"
+	on offer
+		event "Graveyard insights"
+		fail
+
+event "Graveyard insights"
+	planet Alix
+		spaceport `Long ago, the Builders stripped this moon to build the nearby Parii'het space station and produce Ka'het hulls. The Ka'het attacks were limited to the inhabited settlements on Alix; had they also destroyed the mining bases and the unmanned drones, the station would have run out of supplies thousands of years ago.`
```

Comment:
You sure about these? Adding a spaceport means the player is suddenly able to get fuel from landing on these planets just because they got more information from the Remnant.

Reply:
That's what I found out while testing too, but is there really an option other than completely changing the planetary descriptions and removing the old one? While remaining stranded in the Graveyard is definetly nice, until we have something like https://github.com/endless-sky/endless-sky/issues/666 that separates spaceport and automatic refuel this seems like the best course of action, as it isn't that big of a drawback to have the player refuel there

Reply:
> While remaining stranded in the Graveyard is definetly nice, until we have something like #666 that separates spaceport and automatic refuel this seems like the best course of action, as it isn't that big of a drawback to have the player refuel there

I think that being able to suddenly refuel in the Graveyard is a very big drawback compared to what is gained by having expanded descriptions in this way.

Reply:
Changed not to use spaceport but only the normal descriptions